### ISSUE TITLE
Is there any source code?

### ISSUE THREAD

**Issue by username1565** [symbolic/KMSpico#1]
Is there any source code?
ServerService.zip working incorrectly, need to fix and recompile this.

**Comment by LucBerge** [comment]
@symbolic is probably not the owner of the code. **The only one official program** can be found on [this thread](https://forums.mydigitallife.net/threads/kmspico-official-thread.65739/).

The original author provided the following informations : 

> File     : KMSpico_setup.exe
Size     : 3,08 MB
CRC32    : 683a3351
MD5      : a02164371a50c5ff9fa2870ef6e8cfa3
SHA1     : 060614723f8375ecaad8b249ff07e3be082d7f25

To make sure the file on github is the original one, you can compare the two hashes. By downloading the [release](https://github.com/symbolic/KMSpico/releases) and uploading it to a simple online hash tool like [this one](http://onlinemd5.com/).

**Comment by oijdfdg** [comment]
No, the program is obfuscated with dotfuscator and cannot be modified.

**Comment by username1565** [comment]
@rosenishere @LucBerge  I already found an open-source solution - [vlmcsd](https://github.com/Wind4/vlmcsd)
